src/MvvmNano.Core/IPresenter.cs

```diff
@@ -10,6 +10,12 @@ namespace MvvmNano
     public interface IPresenter
     {
         /// <summary>
+        /// Changes the root page to the view of the given View model.
+        /// </summary>
+        /// <typeparam name="TViewModel"></typeparam>
+        void ChangeRootViewModel<TViewModel>() where TViewModel : MvvmNanoViewModel;
```

Comment:
I use the method in the demo to switch between the login screen and the MasterDetail view, you can see the idea behind it there. 

> shouldn't belong into the view model, but the presenter itself

I dont think i get your point here 

Since one could implement this in a custom presenter if needed, it is on you to decide wether or not we should keep this method.

Implementation in a custom presenter for the demo:

	if (page is LoginPage)  
	{  
		Application.SetUpMainPage<LoginViewModel();  
	}  
	if(page is MasterPage){  
		((MvvmNanoMasterDetailApplication)Application).SetUpMasterDetailPage<MasterViewModel>();  
	}